src/Bicep.Core/Parsing/Lexer.cs

```diff
@@ -409,6 +450,45 @@ private void ScanNewLine()
             }
         }
 
+        private TokenType ScanMultilineString()
+        {
+            const int terminatingQuoteCount = 3;
```

Comment:
>terminatingQuoteCount [](start = 22, length = 21)

This constant should be at the class level and shared between the two functions.